Q35/SMBus/smbus.inf

```diff
@@ -0,0 +1,47 @@
+;-------------------------------------------------------------------------------
+; smbusq35.INF
+;
+; Red Hat SM Bus Q35
+;
+; Copyright (c) 2017 Red Hat, Inc.  All rights reserved.
+;
+;-------------------------------------------------------------------------------
+
+[Version]
+Signature = "$Windows NT$"
+Class = System
+ClassGuid = {4D36E97D-E325-11CE-BFC1-08002BE10318}
+Provider = %RedHat%
+DriverVer=04/27/2017,100.0.0.0
+DriverPackageType = PlugAndPlay
+DriverPackageDisplayName = %smbus.DeviceDesc%
+CatalogFile  = smbus.cat
+
+; ======
+; Following two lines are questionable works without them
+DriverPackageType = PlugAndPlay
+DriverPackageDisplayName = %smbus.DeviceDesc%
+
+[Manufacturer]
+%smbus.DeviceDesc% = Models, NTamd64
```

Comment:
Does this .inf really work on 32-bit Windows? I would have expected

`%smbus.DeviceDesc% = Models,NTx86,NTamd64`

here and a

`[Models.NTx86]`

section below. Thanks!